src/__init__.py

```diff
@@ -18090,18 +18082,17 @@ def get_tessdata() -> str:
         message("Tesseract-OCR is not installed")
         return False
 
-    # determine tessdata via iteration over subfolders
-    tessdata = None
-    for sub_response in response.iterdir():
-        for sub_sub in sub_response.iterdir():
-            if str(sub_sub).endswith("tessdata"):
-                tessdata = sub_sub
-                break
-    if tessdata is not None:
+    # search tessdata in folder structure
+    dirname = response[1]  # contains tesseract-ocr installation folder
+    if isinstance(dirname, bytes):  # has probably 'bytes' format
+        dirname = dirname.decode()
```

Comment:
This is ok, but in future we could pass `text=True` to `subprocess.run()` to do this automatically.